### PR TITLE
Add support for NilClass

### PR DESCRIPTION
**Pull request by januszm**
This might be useful for people who also want to set strings
like "Nothing" or "No" to nil values eg. if form boolean input wasn't provided

### CODE REVIEW

**Comment by januszm** [comment]
Alternatively one can always make sure his form inputs are converted to boolean ```!!some_value```. Maybe it's even better way to write confident code, so ... ;)

File: lib/humanize_boolean.rb [hunk 1]
```diff
@@ -24,3 +24,14 @@ def humanize
     I18n.t "boolean.no", :default => "No"
   end
 end
+
+class FalseClass
```
Comment (parabuzzle) [inline lib/humanize_boolean.rb]: shouldn't this be NilClass?

**Comment by parabuzzle** [comment]
Can you also update the tests for this added feature please.

**Comment by januszm** [comment]
Sure @parabuzzle , sorry about that, I was in a bit of rush yesterday. I updated the test and class definition.

**Comment by stevenspiel** [comment]
This would be really helpful. What is the status on merging this in?

**Comment by parabuzzle** [comment]
@stevenspiel I agree it would be helpful. wanna make the tests pass? ;)

**Comment by januszm** [comment]
Oh now I see that I forgot to fix this test:

```ruby
test_human_boolean_in_pirate(TestTranslations) [/Users/janusz/work/humanize_boolean/test/test_translation.rb:12]:
<"Nothing"> expected but was
<"Argh">.
```

**Comment by parabuzzle** [comment]
merged in via https://github.com/parabuzzle/humanize_boolean/pull/3

Thank you for your contribution :+1:

**Comment by parabuzzle** [comment]
pushed the new gem (version 0.0.2) to rubygems.org, it should now be available for use.

**Pull request merged**